src/textarea.rs

```diff
@@ -1304,6 +1306,11 @@ impl<'a> TextArea<'a> {
         }
     }
 
+    /// Sets a flag indicating if the text should be masked with * characters
+    ///
+    pub fn set_password_mode(&mut self, value: bool) {
```

Comment:
```suggestion
    pub fn set_mask(&mut self, mask: Option<char>) {
```

- I'd suggest to allow specifying a character for masking since I'm not sure `*` is always appropriate
- 'password mode' name is too specific. I think it's one of use cases of masking text. Can we give it more generic name `set_mask`?